### PR TITLE
Installer cleanup and consolidation.

### PR DESCRIPTION
**Pull request by charlesbetros**

### CODE REVIEW

File: source/Cosmos.Build.Builder/Tasks/CosmosTask.cs [hunk 1]
```diff
@@ -0,0 +1,313 @@
...
+
+namespace Cosmos.Build.Builder.Tasks
+{
+  /// <summary>
+  /// Cosmos task.
+  /// </summary>
+  /// <seealso cref="Task" />
+  public class CosmosTask : Task
+  {
+    private NugetTask mNugetTask;
```
Comment (jp2masa) [inline source/Cosmos.Build.Builder/Tasks/CosmosTask.cs]: I can't find any reference to `mNugetTask`. Is it needed?

Comment (charlesbetros) [inline source/Cosmos.Build.Builder/Tasks/CosmosTask.cs]: Nope. I meant to remove it.

Comment (jp2masa) [inline source/Cosmos.Build.Builder/Tasks/CosmosTask.cs]: Will you remove it or should I merge this?

**Comment by czhower** [comment]
Matthijs said he would look at it. Or Charles can merge it if he has time. Id like to get it in as soon as we can so we dont end up with a merge conflict. Fortunately we are not working in these areas currently so any conflict should be minimal if any...

**Pull request closed without merging**